magic_the_gathering/phases/main.py

```diff
@@ -0,0 +1,30 @@
+from magic_the_gathering.game_state import GameState
+from magic_the_gathering.phases.base import Phase
+
+
+class MainPhase(Phase):
+    def __init__(self, name: str):
+        super().__init__(
+            name=name,
+            allows_sorcery_speed_when_stack_is_empty=True,
+        )
+
+    def run(self, game_state: GameState) -> GameState:
+        # TODO: Review the implementation of this method
+
+        # Current player can play a land (only 1 per turn)
+        hand_card_index = game_state.current_player.choose_land_to_play()
```

Comment:
Let's keep in mind that a player can play their land after having taken other games actions during the main phase